Libraries/PeriphDrivers/Include/MAX32662/pt.h

```diff
@@ -179,7 +179,7 @@ void MXC_PT_SetPattern(unsigned pts, uint32_t pattern);
 void MXC_PT_EnableInt(uint32_t pts);
 
 /**
- * @brief      Disable interrupts for the pulse trains selected.
+ * @brief      Disable Stop interrupts for the pulse trains selected.
```

Comment:
This goes for other chips and functions as well, could you add a comment explaining that this function is included for compatibility and that users should use MXC_PT_DisableStopInt() instead?

Reply:
Should we add the deprecated attribute for the EnableInt, DisableInt, ClearFlags, and GetFlags functions in favor of the new Stop and Ready functions? Or just leave it as is and add the note? (Assuming we're going forward with using STOP_INTFL/STOP_INTEN instead of INTFL/INTEN)

Reply:
I would think it's best practice to add deprecated? Does adding just cause a warning during compilation if someone tries to use that function?

Reply:
Agreed, add deprecated.  Yes, it counts as a warning.  However, the warning is thrown whenever somebody compiles the file, _not_ when someone uses the function.  So it's not trivial.  We will need some sort of compiler definitions to wrap the deprecated functions in to avoid the warnings every time the PT drivers are compiled.

See [nvic_table.h](https://github.com/Analog-Devices-MSDK/msdk/blob/main/Libraries/PeriphDrivers/Include/MAX32520/nvic_table.h) for an example